Docker Compose v1 deprecation

Last CI workflows are failing with the following error that indicates that `docker-compose` is not available in the GitHub Actions runner environment and thus unable to deploy a dockerized XNAT:
```
Run docker-compose --version
/home/runner/work/_temp/4a332085-bc48-49fe-be94-0c835b3bc4c2.sh: line 1: docker-compose: command not found
Error: Process completed with exit code 127.
```
Based on https://github.com/orgs/community/discussions/134407, Docker Compose v1 has been deprecated and removed from GitHub Actions runners. Official [deprecation notice](https://github.blog/changelog/2024-04-10-github-hosted-runner-images-deprecation-notice-docker-compose-v1/).  

We therefore need to update our workflow to use Docker Compose v2 as suggested [here](https://github.com/orgs/community/discussions/116610).

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -30,7 +30,7 @@ jobs:
           repository: NrgXnat/xnat-docker-compose
           ref: 1.8.1
           path: xnat-docker-compose
-      - run: docker compose --version
+      - run: docker compose version
       - run: python --version
       - run: python -m pip install --upgrade pip
       - name: Install dependencies

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -30,18 +30,18 @@ jobs:
           repository: NrgXnat/xnat-docker-compose
           ref: 1.8.1
           path: xnat-docker-compose
-      - run: docker-compose --version
+      - run: docker compose --version
       - run: python --version
       - run: python -m pip install --upgrade pip
       - name: Install dependencies
         run: pip install -r requirements-dev.txt
       - name: Start xnat-docker-compose
-        run: docker-compose up -d
+        run: docker compose up -d
         working-directory: xnat-docker-compose
       - run: sleep 120
         if: ${{ matrix.python-version == '3.8' }}
       - name: Check xnat-docker-compose
-        run: docker-compose logs --tail=20 xnat-web
+        run: docker compose logs --tail=20 xnat-web
         working-directory: xnat-docker-compose
       - name: create config json (Central/NITRC)
         id: create-json-central

--- a/pyxnat/tests/array_test.py
+++ b/pyxnat/tests/array_test.py
@@ -18,8 +18,8 @@ class ArrayTests(unittest.TestCase):
         of experiments (i.e. MRSessions and PETSessions) and assert it gathers
         them all.
         '''
-        e = self._intf.array.experiments(subject_id='XNAT_S04207').data
-        self.assertEqual(len(e), 2)
+        e = self._intf.array.experiments(subject_id='xnat_S02636').data
+        self.assertEqual(len(e), 1)
 
     @skip_if_no_network
     def test_array_mrsessions(self):
@@ -28,8 +28,8 @@ class ArrayTests(unittest.TestCase):
         list of MRI sessions and assert its length matches the list of
         experiments of type 'xnat:mrSessionData'
         '''
-        mris = self._intf.array.mrsessions(subject_id='XNAT_S04207').data
-        e = self._intf.array.experiments(subject_id='XNAT_S04207',
+        mris = self._intf.array.mrsessions(subject_id='xnat_S02636').data
+        e = self._intf.array.experiments(subject_id='xnat_S02636',
                                          experiment_type='xnat:mrSessionData')
         exps = e.data
         self.assertListEqual(mris, exps)

